### PR TITLE
rename name to appname in logs

### PR DESCRIPTION
**Pull request by hakubo**
## Links

* https://wikia-inc.atlassian.net/browse/XW-1970

@Wikia/x-wing

### CODE REVIEW

File: server/app/lib/logger.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ export function createLogger(loggerConfig) {
 	}
 
 	return bunyan.createLogger({
-		name: 'mercury',
+		appname: 'mercury',
```
Comment (rogatty) [inline server/app/lib/logger.js]: Did you check if it's working? Documentation says that name is required. 
https://github.com/trentm/node-bunyan/blob/master/README.md

Comment (tomasznapieralski) [inline server/app/lib/logger.js]: Jenkins says that it is :D

Comment (rogatty) [inline server/app/lib/logger.js]:
```
/var/lib/jenkins/workspace/Mercury-PR-Checks@3/server/tests/node-qunit.runner.js:45
		throw err;
		      ^
TypeError: options.name (string) is required
    at new Logger (/var/lib/jenkins/workspace/Mercury-PR-Checks@3/www/server/node_modules/bunyan/lib/bunyan.js:306:19)
    at Function.createLogger (/var/lib/jenkins/workspace/Mercury-PR-
```

**Comment by hakubo** [comment]
We don't care about name, this can stay
but we have to add appname so our logstash can understand that

**Pull request closed without merging**